examples/src/main/scala/org/apache/spark/examples/mllib/DatasetExample.scala

```diff
@@ -81,18 +81,18 @@ object DatasetExample {
     println(s"Loaded ${origData.count()} instances from file: ${params.input}")
 
     // Convert input data to SchemaRDD explicitly.
-    val schemaRDD: SchemaRDD = origData
+    val schemaRDD: DataFrame = origData
```

Comment:
minor - but there are a handful of places here where rdd is used in the variable or method argument name, it might be nice to change those.